direct/nn/rim/tests/test_rim.py

```diff
@@ -1,11 +1,12 @@
 # coding=utf-8
 # Copyright (c) DIRECT Contributors
-
 import pytest
 import torch
+import numpy as np
 
-from direct.data.transforms import fft2, ifft2
 from direct.nn.rim.rim import RIM
+from direct.nn.openvino_model import OpenVINOModel
+from direct.data.transforms import fft2, ifft2
```

Comment:
not needed

Reply:
We had this import before changes, but on the line above